experimental/samples/runtime/dotnet/echo-bot/echo-bot/echo-bot.csproj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AspNetCoreHostingModel>OutOfProcess</AspNetCoreHostingModel>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.8" />
+    <PackageReference Include="Microsoft.Bot.Builder.Runtime" Version="4.12.0-daily.preview.20201207.192346.a48b032" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.1.2" />
```

Comment:
I thought I saw the MSTest reference the other day and thought I was going crazy, but I guess not! Do we need this reference? I would assume not?

Reply:
Probably not for a sample?